packages/select/src/useSelect.ts

```diff
@@ -313,7 +313,7 @@ export const useSelect = (props, states: States, ctx) => {
       const option = states.options[i]
       if (states.query) {
         // highlight first options that passes the filter
-        if (!option.props.disabled && !option.props.groupDisabled && option.visible) {
+        if (!option.$props.disabled && !option.$props.groupDisabled && option.visible) {
```

Comment:
We can safely replace `options.$props.dsiabled` by `options.dislabed` I think?

```suggestion
        if (!option.disabled && !option.groupDisabled && option.visible) {
```